### PR TITLE
Jetpack Visual Refresh: "At A Glance" Dashboard adjustments

### LINKED ISSUE #29832: Jetpack Plugin Visual Refresh: At A Glance Dashboard
Visual refresh for the Jetpack Plugin At a Glance dashboard. 

- Most borders removed to reduce business of page. Padding, type size, color, and buttons adjusted.
- Main navigation & footer updated to match all our other products.
- All cards styled to resemble cards in My Jetpack and inactive cards are easier to identify.
- Mini notices at the top of Dashboard and Settings cards now consistent with Emerald.
- Security upsell gets a special highlight

For more images and details: p1HpG7-lx8-p2#dashboard

Image

### PR DESCRIPTION
**Pull request by sergeymitr**
Fixes #29832

## Proposed changes:
* Minor styling adjustments for the "At A Glance" page.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion
p1HpG7-lx8-p2

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
Make sure the page looks according to the design, and all the issues (#29832) were addressed.

### CODE REVIEW

**Comment by jeffgolenski** [comment]
Outstanding, as always @sergeymitr. As mentioned in slack, I found a couple more mobile issues that just need to be cleaned up. Here are a few screenshots:


IMG_8111

- [x] For small screens, can the install and upgrade buttons remain flush to the left and line up with the text above it?

IMG_8113

- [x] The mobile CTA needs the "download on the app store" label fixed. Check the mobile view on /stats for the proper view

IMG_8112

- [x] Manage your clients with ease upsell: Fix button and close icon. The button can drop to the next line!

File: projects/plugins/jetpack/_inc/client/at-a-glance/stats/index.jsx [hunk 1]
```diff
@@ -154,30 +154,26 @@ export class DashStats extends Component {
 
 	dismissCard = () => {
 		this.setState( { emptyStatsDismissed: true } );
-		this.props.updateOptions( { dismiss_empty_stats_card: true } );
+		// this.props.updateOptions( { dismiss_empty_stats_card: true } );
```
Comment (nateweller) [inline projects/plugins/jetpack/_inc/client/at-a-glance/stats/index.jsx]: Potentially left over from testing? Commented out code here and on line 214.

Comment (sergeymitr) [reply projects/plugins/jetpack/_inc/client/at-a-glance/stats/index.jsx]: Good catch.
We should be good to remove that now.

**Comment by jeffgolenski** [comment]
Changes look good, thanks @sergeymitr !

**Review by jeherve — APPROVED** [review]
This tests well for me. 🚢

**Comment by zinigor** [comment]
Cherry picked in https://github.com/Automattic/jetpack/commit/f0bca94fc62001caf1d4481cc954e947e498b372.

**Pull request merged**